frontend/mobilityai_react/App/Screens/PatientData.js

```diff
@@ -55,44 +65,68 @@ export default class PatientData extends Component {
 
     pieColour(i) {
 
-    }
+    };
 
+    componentDidMount(){
+        //TODO: Remove hard coded dates/times once date picker works
+        // let startTime = new Date(Date.UTC(2018, 11, 11, 0, 0, 0, 0)).getTime();
+        // let endTime = new Date(Date.UTC(2018, 11, 11, 23, 0, 0, 0)).getTime();
 
+        let startTime = new Date(Date.UTC(2019, 1, 9, 0, 0, 0, 0)).getTime();
+        let endTime = new Date(Date.UTC(2019, 1, 9, 23, 0, 0, 0)).getTime();
+
+        const { navigation } = this.props;
+        const id = navigation.getParam('id');
+
+        GetPatientActivities(startTime, endTime, id).then((activitiesJson) => {
+            if (activitiesJson === undefined) {
+                this.setState({error: 'Error retrieving patient activity data'});
+            }
+            console.log('\n\n' + activitiesJson);
+            this.setState({movementPercentages: activitiesJson});
+        });
+    };
 
     render() {
+        if (this.state.error) {
+            return (
+                <View>
```

Comment:
We should make this a real error component that we can reuse across the app.

Reply:
@rebeccatran can do that for next time. #122